doc/source/tune/_tutorials/tune-pytorch-lightning.rst

```diff
@@ -282,14 +288,24 @@ Adding checkpoints to the PyTorch Lightning module
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 First, we need to introduce
-another callback to save model checkpoints:
+another callback to save model checkpoints. Since Tune requires a call to
+``tune.report()`` after creating a new checkpoint to register it, we will use
+a combined reporting and checkpointing callback:
 
-.. literalinclude:: /../../python/ray/tune/examples/mnist_pytorch_lightning.py
-   :language: python
-   :start-after: __tune_checkpoint_callback_begin__
-   :end-before: __tune_checkpoint_callback_end__
+.. code-block:: python
+
+    from ray.tune.integration.pytorch_lightning import TuneReportCheckpointCallback
+    callback = TuneReportCheckpointCallback(
+        metrics={"loss": "val_loss", "mean_accuracy": "val_accuracy"},
+        filename="checkpoint",
+        on="validation_end")
+
+The ``checkpoint`` value is the name of the checkpoint file within the
+checkpoint directory.
 
-We also include checkpoint loading in our training function:
+We also include checkpoint loading in our training function. The name
+of the checkpoint has to coincide with the one we passed to the
+``TuneCheckpointCallback``:
```

Comment:
let's mark it as internal

Reply:
(TuneCheckpointCallback is no longer public right?)

Reply:
Updated the tutorial